src/corehost/cli/dll/CMakeLists.txt

```diff
@@ -26,5 +34,5 @@ endif()
 
 add_definitions(-DCOREHOST_MAKE_DLL=1)
 
-add_library(clihost SHARED ${SOURCES})
+add_library(hostimpl SHARED ${SOURCES})
```

Comment:
Can I suggest a minor change to `corehostimpl`, `coreclrhost`, `clrhostimpl`, etc.? Basically just to make it clear that this is related to the CLR hosting specifically. As a developer, I think that having a random dylib inserted into my build output would be a little surprising to me but if it was named something that made it clear it was used to launch the CLR that would reassure me :)